server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/TransportGetSnapshotsAction.java

```diff
@@ -59,7 +66,7 @@ public TransportGetSnapshotsAction(TransportService transportService, ClusterSer
                                        ThreadPool threadPool, SnapshotsService snapshotsService, ActionFilters actionFilters,
                                        IndexNameExpressionResolver indexNameExpressionResolver) {
         super(GetSnapshotsAction.NAME, transportService, clusterService, threadPool, actionFilters,
-            GetSnapshotsRequest::new, indexNameExpressionResolver);
+                GetSnapshotsRequest::new, indexNameExpressionResolver);
```

Comment:
Revert this formatting change?

Reply:
https://github.com/elastic/elasticsearch/pull/42090/commits/be898f4f18a5b7c3c5f872abe2271e0900c0da01